PytestDeprecationWarning: Direct construction of Item has been deprecated, please use Item.from_parent.

```
.tox/py/lib/python3.8/site-packages/pluggy/hooks.py:286: in __call__
    return self._hookexec(self, self.get_hookimpls(), kwargs)
.tox/py/lib/python3.8/site-packages/pluggy/manager.py:93: in _hookexec
    return self._inner_hookexec(hook, methods, kwargs)
.tox/py/lib/python3.8/site-packages/pluggy/manager.py:84: in <lambda>
    self._inner_hookexec = lambda hook, methods, kwargs: hook.multicall(
.tox/py/lib/python3.8/site-packages/pytest_pydocstyle.py:42: in pytest_collect_file
    return Item(path, parent, parser)
.tox/py/lib/python3.8/site-packages/_pytest/nodes.py:95: in __call__
    warnings.warn(NODE_USE_FROM_PARENT.format(name=self.__name__), stacklevel=2)
E   pytest.PytestDeprecationWarning: Direct construction of Item has been deprecated, please use Item.from_parent.
E   See https://docs.pytest.org/en/stable/deprecations.html#node-construction-changed-to-node-from-parent for more details.
```

--- a/.github/workflows/pythonpackage.yml
+++ b/.github/workflows/pythonpackage.yml
@@ -8,7 +8,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: [3.5, 3.6, 3.7, 3.8]
+        python-version: [3.6, 3.7, 3.8]
 
     steps:
     - uses: actions/checkout@v2

--- a/pyproject.toml
+++ b/pyproject.toml
@@ -25,6 +25,6 @@ deps =
   pytest-cov
   pytest-flake8
   pytest-mypy
-  pytest-pydocstyle==2.0.0
+  pytest-pydocstyle>=2.1.0
 extras = testing
 """

--- a/setup.cfg
+++ b/setup.cfg
@@ -23,7 +23,7 @@ install_requires =
   pandas
   scikit-learn>=0.19.0
 packages = find:
-python_requires = >=3.5
+python_requires = >=3.6
 
 [options.extras_require]
 testing =

